railties/lib/rails/generators/app_base.rb

```diff
@@ -405,11 +405,19 @@ def run_bundle
       end
 
       def run_webpack
-        if webpack_install?
-          rails_command "webpacker:install"
-          if options[:webpack] && options[:webpack] != "webpack"
-            rails_command "webpacker:install:#{options[:webpack]}"
-          end
+        return unless webpack_install?
+
+        unless bundle_install?
+          say <<~EXPLAIN
+            Ignoring `webpacker:install` because option that skips `bunde install` was enabled
+            `rails webpacker:install` requires `bundle install` to be ran first
```

Comment:
```suggestion
            Skipping `rails webpacker:install` because `bundle install` was skipped.
            To complete setup, you must run `bundle install` followed by `rails webpacker:install`.
```

Or, if we want to be more explicit:

```suggestion
            Skipping `rails webpacker:install` because `--skip-bundle` or `--skip-gemfile` was specified.
            To complete setup, you must run `bundle install` followed by `rails webpacker:install`.
```